src/hub/dataload/sources/taxonomy/uploader.py

```diff
@@ -9,53 +10,70 @@ class TaxonomyNodesUploader(uploader.BaseSourceUploader):
     main_source = "taxonomy"
     name = "nodes"
 
-    def load_data(self,data_folder):
-        nodes_file = os.path.join(data_folder,"nodes.dmp")
+    def load_data(self, data_folder):
+        nodes_file = os.path.join(data_folder, "nodes.dmp")
         self.logger.info("Load data from file '%s'" % nodes_file)
         return parse_refseq_nodes(open(nodes_file))
 
     @classmethod
     def get_mapping(klass):
         return {
-                "rank": {
-                    "type": "text"
-                    },
-                "taxid": {
-                    "copy_to" : ["all"],
-                    "type": "long"
-                    },
-                "parent_taxid": {
-                    "type": "long"
-                    },
-                }
+            "rank": {
+                "type": "text"
+            },
+            "taxid": {
+                "copy_to": ["all"],
+                "type": "long"
+            },
+            "parent_taxid": {
+                "type": "long"
+            },
+        }
 
 
 class TaxonomyNamesUploader(uploader.BaseSourceUploader):
 
     main_source = "taxonomy"
     name = "names"
     __metadata__ = {
-            "mapper" : 'has_gene',
-            }
+        "mapper": 'has_gene',
+    }
 
-    def load_data(self,data_folder):
-        names_file = os.path.join(data_folder,"names.dmp")
+    def load_data(self, data_folder):
+        names_file = os.path.join(data_folder, "names.dmp")
         self.logger.info("Load data from file '%s'" % names_file)
         return parse_refseq_names(open(names_file))
 
     @classmethod
     def get_mapping(klass):
         return {
-                "scientific_name": {
-                    "copy_to" : ["all"],
-                    "type": "text"
-                    },
-                "common_name": {
-                    "copy_to" : ["all"],
-                    "type": "text"
-                    },
-                "genbank_common_name": {
-                    "type": "text"
-                    }
+            "scientific_name": {
+                "copy_to": ["all"],
+                "type": "text",
+                "fields": {
+                        "raw": {
+                            "type": "keyword",
+                            "normalizer": "keyword_lowercase_normalizer"
```

Comment:
you may need to test if this normalizer will return original case or lower-cased facet values. If still lower-cased, we might need to add a `raw2` sub-field without this normalizer.

Reply:
You're correct, it will return lowercased values in aggregations. Updated with a `raw2` field.